_layouts/base.html

```diff
@@ -10,6 +10,39 @@
         {% if page.description %}<meta name="description" content="{{ page.description }}">{% endif %}
         <link href="{{ site.baseurl }}/css/custom.css" rel="stylesheet" type="text/css">
         <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
+        <!--Below is the schema annotation for blogposts-->
+        {% if page.layout == 'post' %}
+        <script type="application/ld+json">
+          {
+            "@context": "https://schema.org",
+            "@type": "BlogPosting",
+            "mainEntityOfPage": {
+              "@type": "WebPage",
+              "@id": "{{ site.url }}/posts/"
+              },
+            "headline": "{{ page.title }}",
+            "description": "{{ page.excerpt | strip_html | strip_newlines | truncate: 200 }}",
+            "image": "{{ page.image }}",
+            "author": {
+                  "@type": "Person",
```

Comment:
Could you make here a `for` loop to add each author as an `author` property?

Reply:
Smart. 💯 All done.

Reply:
I gave the authors unique urls. Please let me know if you would rather I loop from the top to give each person a unique author property...

Reply:
Yes I think it would be better to give each author their own property

Reply:
Cool. I'll update that right away...